cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -2562,7 +2635,7 @@ pub(crate) mod tests {
             },
             InstalledPackage {
                 install_id: bar_iid,
-                rel_path: bar_descriptor.pkg_path,
+                rel_path: bar_pkg_path,
```

Comment:
nonblocking: I don't see any test coverage for any of these changes. I think we could add some locking tests with flake descriptors. We can defer that until we actually implement locking flake descriptors though

Reply:
yep will be doing this with the locking pr